enterprise/frontend/src/metabase-enterprise/sandboxes/components/EditSandboxingModal/EditSandboxingModal.unit.spec.tsx

```diff
@@ -130,6 +136,61 @@ describe("EditSandboxingModal", () => {
           screen.getByText("Restrict access to this table"),
         ).toBeInTheDocument();
 
+        expect(
+          screen.getByText("Filter by a column in the table"),
+        ).toBeInTheDocument();
+
+        expect(
+          await screen.findByText(
+            "Use a saved question to create a custom view for this table",
+          ),
+        ).toBeInTheDocument();
+
+        expect(screen.getByRole("button", { name: "Save" })).toBeDisabled();
+
+        await userEvent.click(await screen.findByText("Pick a column"));
+        await userEvent.click(await screen.findByText("ID"));
+
+        await userEvent.click(screen.getByText("Pick a user attribute"));
+        await userEvent.click(await screen.findByText("foo"));
+
+        await userEvent.click(screen.getByText("Save"));
+
+        await waitFor(() =>
+          expect(onSave).toHaveBeenCalledWith({
+            attribute_remappings: {
+              foo: [
+                "dimension",
+                ["field", PEOPLE.ID, { "base-type": "type/BigInteger" }],
+                { "stage-number": 0 },
+              ],
+            },
+            card_id: null,
+            group_id: 1,
+            table_id: PEOPLE_ID,
+          }),
+        );
+      });
+
+      it("should not allow sandboxing with a question if that feature is not enabled", async () => {
```

Comment:
This test passes whether you pass features or not. I'll see if I can update it 👍